src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -1,28 +1,75 @@
 package nextstep.subway.line.ui;
 
+import static org.springframework.http.MediaType.*;
+import static org.springframework.http.ResponseEntity.*;
+
+import nextstep.subway.line.application.DuplicationKeyException;
 import nextstep.subway.line.application.LineService;
 import nextstep.subway.line.dto.LineRequest;
 import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.line.exception.NotExistsLineException;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.net.URI;
+import java.util.List;
 
 @RestController
 @RequestMapping("/lines")
 public class LineController {
     private final LineService lineService;
+    private Long id;
 
     public LineController(final LineService lineService) {
         this.lineService = lineService;
     }
 
-    @PostMapping
-    public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
+    @PostMapping(consumes = {APPLICATION_JSON_VALUE})
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
```

Comment:
HTTP 요청시 미디어 타입을 application/json으로 설정하여 요청하다고 가정하였을 때,
컨틀롤러에서는 @RequestBody를 사용하고 클래스 타입이 객체 또는 HashMap 이면 
HTTP 메시지 컨버터(MappingJackson2HttpMessageConverter)가 설정되기 때문에 생략해도 consume은 생략해도 괜찮을 것 같습니다 :)

궁금하시다면 MappingJackson2HttpMessageConverter 클래스의 canRead가 어떻게 동작하는지 참고하시면 좋을 것 같아요 :)
```suggestion
    @PostMapping
    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
```

Reply:
제한 하고 싶다는 생각으로 접근 했는데 이것도 확인해봐야겠네요 !!